tests/system/small/operations/test_datetimes.py

```diff
@@ -16,16 +16,22 @@
 import pytest
 
 import bigframes.series
-from tests.system.utils import assert_series_equal
+from tests.system.utils import assert_series_equal, skip_legacy_pandas
 
 DATETIME_COL_NAMES = [("datetime_col",), ("timestamp_col",)]
+DATE_COLUMNS = [
+    ("datetime_col",),
+    ("timestamp_col",),
+    ("date_col",),
+]
 
 
 @pytest.mark.parametrize(
     ("col_name",),
-    DATETIME_COL_NAMES,
+    DATE_COLUMNS,
 )
-def test_day(scalars_dfs, col_name):
+@skip_legacy_pandas
+def test_dt_day(scalars_dfs, col_name):
     if pd.__version__.startswith("1."):
         pytest.skip("Pyarrow datetime objects not support in pandas 1.x.")
```

Comment:
The @skip_legacy_pandas seems already skipping pandas ver 1.x,  I think this "if" will always be false? Same with test_dt_date

Reply:
good catch, removed the redundant checks